README.md

```diff
@@ -8,13 +8,27 @@ This repository is matched with downloadable input data so that it will run out
 
 Copy the `.env.example` file to `.env` and customise the paths as you need.
 
+In order to download the GFAS emissions data,
+credentials for the Copernicus Atmospheric Data Store (ADS) API are required.
+Instructions for registering for the ADS API and setting up the credentials are provided
+at in [ADS Docs](https://ads.atmosphere.copernicus.eu/api-how-to).
+
+After creating an account and obtaining the credentials,
+you must also accept the 
+[license conditions for the data](https://ads.atmosphere.copernicus.eu/cdsapp/#!/terms/licence-to-use-copernicus-products)
```

Comment:
I had to accept a license on the specific data set which is being downloaded, which was here: https://ads.atmosphere.copernicus.eu/cdsapp#!/dataset/cams-global-fire-emissions-gfas?tab=form

Reply:
Thanks I think that is an alternative way of achieving the same result. I've added a sentence that should enable people to verify that they have accepted the terms of use